frontend/src/components/APRRateDisplay.js

```diff
@@ -237,29 +258,54 @@ class APRRateDisplay extends Component {
       />
     )
   }
+  
   getCSV() {
-      const { formDataReducer, contNewLoan, changedFormData } = this.props
-    	let csv = [['Period Number,Date,Days,Principal Disbursed,Principal Paid,Balance,Interest Paid,Fees Paid,Insurance Paid,Taxes Paid,Security Deposit,Interest Paid on Security,Deposit Withdrawal,Deposit Balance,Total Cashflow\n']];
-      let i;
-      let j;
-      let row = "";
-      for (j = 0; j < 13; j++) {
-      for (i = 0; i < 15; i++) {
-        row += formDataReducer.original_repayment_schedule[i][j] + ',';
-      }
-      row+='\n';
-      csv.push(row);
-      row = '';
-      }
-      let blob = new Blob(csv,{type: 'text/csv;charset=utf-8;'});
-      let url = URL.createObjectURL(blob);
-      let pom = document.createElement('a');
-      pom.href = url;
-      pom.setAttribute('download', 'output.csv');
-      pom.click();
+    const { formDataReducer } = this.props
+    let csv = [['Period Number,Date,Days,Principal Disbursed,Principal Paid,Balance,Interest Paid,Fees Paid,Insurance Paid,Taxes Paid,Security Deposit,Interest Paid on Security,Deposit Withdrawal,Deposit Balance,Total Cashflow\n']];
+    for (let j = 0; j < 13; j++) {
+    let row = "";
+        for (let i = 0; i < 15; i++) {
+	    row += formDataReducer.original_repayment_schedule[i][j] + ',';
+	}
+    row+='\n';
```

Comment:
indent the variables row in the first for loop on the same line as the nested for loop